Snippets/Snippets_6/UpgradeGuides/5to6/CustomErrorHandlingBehaviour.cs

```diff
@@ -0,0 +1,56 @@
+namespace Snippets6.UpgradeGuides._5to6
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.Pipeline;
+
+    #region 5to6-customerrorhandlingbehaviour
+    class CustomErrorHandlingBehaviour : Behavior<ITransportReceiveContext>
+    {
+        public override async Task Invoke(ITransportReceiveContext context, Func<Task> next)
+        {
+            try
+            {
+                await next().ConfigureAwait(false);
```

Comment:
@SimonCropp - I tried to take a look at the some of the Pipeline samples in v6, they all use this. I am not sure how you want to change this.
See: https://github.com/Particular/docs.particular.net/blob/master/samples/pipeline/stream-properties/Version_6/Shared/StreamSendBehavior.cs#L62